rmgpy/data/kinetics/family.py

```diff
@@ -134,6 +134,29 @@ def __reduce__(self):
                                    self.reverse,
                                    ))
 
+    def __repr__(self):
+        """
+        Return a string representation that can be used to reconstruct the
+        object.
+        """
+        string = 'TemplateReaction('
+        if self.index != -1: string += 'index={0:d}, '.format(self.index)
```

Comment:
It's the default value, so it should still create the same object. It's possible to print out, but not necessary.